src/test_utils/public/stub_index_pattern.js

```diff
@@ -29,12 +32,13 @@ import { fieldFormats } from 'ui/registry/field_formats';
 
 export default function () {
 
-  function StubIndexPattern(pattern, timeField, fields) {
+  function StubIndexPattern(pattern, getConfig, timeField, fields) {
```

Comment:
This was no longer consistent with the actual `IndexPattern` class, and is only used directly in a few places, so I updated the args